x-pack/test/functional/page_objects/gis_page.ts

```diff
@@ -296,10 +296,27 @@ export class GisPageObject extends FtrService {
     };
   }
 
+  // This method is also used by upgrade testing which is not part of PR testing
+  // Please keep in mind when udpating, removing or adding to this method
+  // upgrade needs to be tested too
+  async clearLegendTooltip() {
+    const isTooltipOpen = await this.testSubjects.exists(`layerTocTooltip`, { timeout: 5000 });
+    if (isTooltipOpen) {
+      await this.testSubjects.click(`layerTocTooltip`);
+      await this.common.sleep(1000);
```

Comment:
What is this sleep waiting for? Can this be removed?

Reply:
Mind adding a comment to timeout about waiting for tooltip close animation?

Reply:
Comment added